packages/react/src/main-nav/NavListItem.tsx

```diff
@@ -81,18 +75,12 @@ export function NavListItem({
 						},
 
 						// Focus styles
-						'&:focus': {
```

Comment:
All this code was doing was making sure the outline wasn't being cut off by the border-bottom psuedo element. That's now been cleaned up